README.rst

```diff
@@ -51,72 +55,123 @@ Build and tests status
 |Develop| |devel-cov|  | |devel-posix|   | |devel-win|  |
 +-------+--------------+-----------------+--------------+
 
+Documentation Build
+-------------------
 
-Quick Start
-===========
++--------+--------------+
+|Version | **RTD Build**|
++========+==============+
+| Latest | |docs-rtd|   |
++--------+--------------+
 
-Install Python 2.7 then download and extract the latest ScanCode release from
-https://github.com/nexB/scancode-toolkit/releases/ 
 
-Then run ``./scancode -h`` for help.
+Documentation
+=============
+
+The ScanCode documentation is hosted at `scancode-toolkit.readthedocs.io <https://scancode-toolkit.readthedocs.io/en/latest/>`_.
+
+If you are new to Scancode, start `here <https://scancode-toolkit.readthedocs.io/en/latest/getting-started/newcomer.html>`_.
+
+Other Important Documentation Pages:
+
+- A `Synopsis <https://scancode-toolkit.readthedocs.io/en/latest/cli-reference/synopsis.html>`_ of ScanCode Command Line Options
+- Tutorials on `How to Run a Scan <https://scancode-toolkit.readthedocs.io/en/latest/tutorials/how_to_run_a_scan.html>`_ and `How to Visualize Scan results <https://scancode-toolkit.readthedocs.io/en/latest/tutorials/how_to_visualize_scan_results.html>`_
+- An exhaustive List of `All Available Options <https://scancode-toolkit.readthedocs.io/en/latest/cli-reference/list-options.html>`_
+- Documentation on `Contributing to Code Development <https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html>`_
+- Documentation on `Plugin Architecture <https://scancode-toolkit.readthedocs.io/en/latest/plugins/plugin_arch.html>`_
+- `FAQ <https://scancode-toolkit.readthedocs.io/en/latest/misc/faq.html>`_
+
+See also https://aboutcode.org for related companion projects and tools.
 
 
 Installation
 ============
 
-Pre-requisites:
+There are 3 main ways you can `install ScanCode <https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html>`_.
```

Comment:
Would it make sense to move the bulk of this in an INSTALL.rst file instead? and only keep the simple happy installation path here?

Reply:
@pombredanne could you elaborate.

Reply:
Okay right. That would definitely be better IMHO, the README being smaller and more concise. Should I do that?  In that case I'd only link to the installation documentation right?

Reply:
Done!